clc/cmd/command_context.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/mk"
+)
+
+type CommandContext struct {
+	Cmd           *cobra.Command
+	Vpr           *viper.Viper
+	stringValues  map[string]*string
```

Comment:
Just wondering. What's the value of `*string` or same for other primitives in the map?

Reply:
Using primitives would copy their values, so Cobra wouldn't be able to change them.